packages/html-elements/src/css/units.ts

```diff
@@ -0,0 +1,11 @@
+import { PixelRatio, Platform } from 'react-native';
+
+export function rem(pixels: number): number | string {
```

Comment:
`pixels` isn't the right variable name. If someone wanted entered 16 "pixels" (which is 1rem, usually), they would instead get 16rem, which is a huge difference. I think renaming `pixels` to `rem/em`, `units` or something similar would clear up any confusion about how to use this.